assets/runtime/config/gitlabhq/gitlab.yml

```diff
@@ -70,6 +70,17 @@ production: &base
     email_display_name: {{GITLAB_EMAIL_DISPLAY_NAME}}
     email_reply_to: {{GITLAB_EMAIL_REPLY_TO}}
     email_subject_suffix: '{{GITLAB_EMAIL_SUBJECT_SUFFIX}}'
+    email_smime:
+      #start-email-smime
```

Comment:
```suggestion

```

Reply:
`#start-email-smime` moved on top of `email_smime`